RefApp/Extension/Date+Extension.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  Date+Extensions.swift
+//  iOSReferenceApp
+//
+//  Created by Fredrik Sjöberg on 2017-07-10.
+//  Copyright © 2017 emp. All rights reserved.
+//
+
+
+import Foundation
+
+extension Date {
```

Comment:
We could possibly include this in the SDK for reusability. Would require unit tests etc, but worth considering later on

Reply:
Yes, we can add this to the SDK in future with proper unit tests 👍